test/models/vote_test.rb

```diff
@@ -0,0 +1,53 @@
+require "test_helper"
+
+describe Vote do
+  let(:vote) { votes(:user1_book1) }
+
+  it "must belong to a user" do
```

Comment:
You should be testing the uniqueness constraint here! Specific test cases I'd want to see:

- A user can have votes for two different works
- A work can have votes from two different users
- A user cannot vote for the same work twice